### PR TITLE
remove back-off exponential test and adjust timing

### PR DESCRIPTION
**Pull request by samsabed**
remove the problematic e2e test ".. timer increase exponentially", and adjust the timing on the "..timer reset on image update"

### CODE REVIEW

**Comment by ixdy** [comment]
@k8s-bot ok to test

File: test/e2e/pods.go [hunk 1]
```diff
@@ -828,39 +828,6 @@ var _ = Describe("Pods", func() {
 		}
 	})
 
-	It("should have their container restart back-off timer increase exponentially", func() {
```
Comment (lavalamp) [inline test/e2e/pods.go]: If you plan to revive, perhaps better to mark as pending? (It -> PIt)

Comment (dchen1107) [inline test/e2e/pods.go]: I think I am ok to remove this test since there are other tests to cover Container restart backoff and image backoff.

**Comment by samsabed** [comment]
@dchen1107 do you think the timeout is somehow related to the patch ?

**Comment by dchen1107** [comment]
No, shippable failure has nothing to do with you patch here.

**Comment by dchen1107** [comment]
Ok, I kick out the re-run here. LGTM

**Comment by wojtek-t** [comment]
@k8s-bot unit test this please

**Comment by samsabed** [comment]
Thanks

**Pull request merged**